analyzer_test.go

```diff
@@ -18,7 +18,7 @@ func TestDeriveManifestAndLock(t *testing.T) {
 	defer h.Cleanup()
 
 	h.TempDir("dep")
-	golden := "analyzer/manifest.json"
+	golden := "analyzer/manifest.toml"
```

Comment:
FWIW, being a PC guy, I've found Go does a startingly good job of using '/' in a platform-independent manner  in path strings.  But `filepath.Join` is still likely more robust.